### ISSUE TITLE
Exception logging

### ISSUE THREAD

**Issue by JamesMcIntosh** [pavlospt/RxFile#7]
I notice that you have hard coded your error messages with line numbers in RxFile.java i.e.

```
logError("Exception: " + e.getMessage() + " line: 123");
e.printStackTrace();
```

Is there any reason you are not just using the standard error logging which will print out a stack with the appropriate line number?

```
public static final String TAG = RxFile.class.getName();
...
Log.e(TAG, "Error description", e);
```

**Comment by pavlospt** [comment]
I did not quite understand what you are asking.

**Comment by JamesMcIntosh** [comment]
I was just unsure as to why you chose to do the error log messages without leveraging the functionality of the logging framework, normally you would use the syntax in my example above.

Generally when logging exceptions you should let the logging framework handle printing a stack trace. The log message from the framework will also give you the line numbers so you should never need to manually put line numbers into a logging statement because as soon refactor the code your line numbers become wrong. If you have put the line numbers to work with obfuscated code then you should learn how to use the source maps correctly or improve your error messages.

**Comment by pavlospt** [comment]
@JamesMcIntosh Yes, I totally agree, it is just that when I was creating it I did not focus so much on the logging part. I can fix it and upload a new version :)

**Comment by pavlospt** [comment]
@JamesMcIntosh this has finally been fixed after all those years in version 1.8 :) Sorry for taking too long for it!

**Comment by JamesMcIntosh** [comment]
Nice!
